MCprep_addon/MCprep_resources/Languages/mcprep.pot

```diff
@@ -0,0 +1,523 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
Calling out this is not a descriptive title :)

Reply:
(just realized this now) POT files are templates, so this is meant to be a placeholder for translators